tests/test_utils_dict_support.py

```diff
@@ -119,12 +137,6 @@ def test_merging_dicts_with_lists():
     assert_semantically_equal(d, {'1': ['', '2'], '2': [], '3': ['1']})
 
 
-def test_merging_dicts_with_lists_with_overlapping_values_returns_list_with_unique_values():
```

Comment:
Yeah, i had enough sense to know how it works but not how it could yield bad results. And didn't feel weird to me at the time to covert a list to set and back to list for some reason :shrug: